rats-tls/dist/rpm/rats_tls.spec.in

```diff
@@ -51,12 +52,12 @@ if [ ! -d "$SGX_SDK" ]; then
         exit 1
 fi
 
-pushd %{name}
+pushd %{source_dir}
```

Comment:
May I ask why change `name` to `source_dir`?

Reply:
Here build script needs to record the build path and popd it once cmd completed and the build path(rats-tls) is a constant string. But %{name} will be alternative with real package name according to different mode. That's why need use other available to record the constant path.